libjcat/jcat-self-test.c

```diff
@@ -880,7 +880,7 @@ jcat_context_verify_item_target_func(void)
 
 	/* set up context */
 	jcat_context_set_keyring_path(context, "/tmp");
-	pki_dir = g_test_build_filename(G_TEST_BUILT, "pki", NULL);
+	pki_dir = g_test_build_filename(G_TEST_BUILT, "pki", "self", NULL);
```

Comment:
why `self`? I'm a bit confused why we ned the extra dir.

Reply:
An alternative to keep the same directory structure would be to use `jcat_context_add_public_key` and the single key instead of the directory.

Reply:
Here's the alternative: https://github.com/hughsie/libjcat/pull/89